pkg/mocks/repository/uploadProfilePictureManager.go

```diff
@@ -0,0 +1,23 @@
+package repository
+
+import (
+	"context"
+
+	"github.com/stretchr/testify/mock"
+)
+
+type ProfilePicture struct {
+	mock.Mock
+}
+
+func (p *ProfilePicture) GetUploadUrl(ctx context.Context, ext string, userID string) (string, error) {
+	outputs := p.Mock.Called(ctx, ext, userID)
+
+	uploadUrl := outputs.String(0)
+	err := outputs.Error(1)
+
+	if uploadUrl != "" {
+		return uploadUrl, err
+	}
+	return "", err
+}
```

Comment:
```suggestion
func (p *ProfilePicture) GetUploadUrl(ctx context.Context, ext string, userID string) (string, error) {
	args := p.Mock.Called(ctx, ext, userID)

	return args.String(0), args.Error(1)
}
```

Reply:
I feel the initial code provides more readability

Reply:
Sure, you can capture the return value in variables first, but `if uploadUrl != ""` doesn't seem to do anything different in the two branches, right?

Reply:
looking at it now,we can have the return statement return uploadUrl and err since the zero value would still be an empty string

Reply:
If the error that is returned it not `nil`, then the value of the string should not be considered by the caller.  
So yes.

Reply:
in mocking we don't write logic (when using testify pkg, if mocking was manual, we should write logic but not needed here)
so here we only return the passes value by mock expectation.